clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -400,10 +400,6 @@ def warn_hlsl_langstd_minimal :
           "recommend using %1 instead">,
   InGroup<HLSLDXCCompat>;
 
-def err_hlsl_semantic_missing : Error<"semantic annotations must be present "
-                                      "for all input and outputs of an entry "
-                                      "function or patch constant function">;
-
```

Comment:
We do still need some diagnostic to cover if we have a missing semantic on a parameter or field right?